lib/src/components/NxTransferList/NxTransferList.tsx

```diff
@@ -159,7 +159,7 @@ export default function NxTransferList<T extends string | number>(props: Props<T
           }
           <div className="nx-transfer-list__item-list">
             { visibleSelectedItems.map(
-                i => <TransferListItem key={i.id} checked={true} onChange={onChange} { ...i } />)
+                i => <TransferListItem key={i.id} checked={true} onChange={onChange} {...i} />)
```

Comment:
A few minor extraneous changes here

Reply:
ops! For future reference, is the style to have extra space in between brackets when spread is use?
This is fixed here:
8df61c7ba4c2ad10415a869de9993da4bfe5b9af

Reply:
> For future reference, is the style to have extra space in between brackets when spread is use?

Strictly speaking eslint is our source of truth there.  It wasn't complaining so your change wasn't wrong just extraneous.  That said, I personally prefer to have these spaces there within curly braces but not within square braces.  Be aware that other projects at Sonatype have somewhat different formatting rules though.